test/Microsoft.SqlTools.ServiceLayer.Test.Common/TestAzureBlobConnectionService.cs

```diff
@@ -54,13 +54,19 @@ internal static AzureBlobConnectionSetting InitAzureBlobConnectionSetting()
                 AzureBlobConnectionSetting settings = new AzureBlobConnectionSetting();
                 settings.AccountKey = Environment.GetEnvironmentVariable(Constants.AzureStorageAccountKey);
                 settings.AccountName = Environment.GetEnvironmentVariable(Constants.AzureStorageAccountName);
-                settings.BlobContainerUri = Environment.GetEnvironmentVariable(Constants.AzureBlobContainerUri); 
-                Console.WriteLine("Azure Blob Connection Settings loaded successfully");
+                settings.BlobContainerUri = Environment.GetEnvironmentVariable(Constants.AzureBlobContainerUri);
+
+                if (String.IsNullOrWhiteSpace(settings.AccountName) || String.IsNullOrWhiteSpace(settings.AccountKey) || String.IsNullOrWhiteSpace(settings.BlobContainerUri))
+                {
+                    throw new InvalidOperationException("Azure Blob connection settings are not set, but are required for this test.");
```

Comment:
Would suggest adding a link to instructions on setting these up too

Reply:
Added.  I'll add a link to that page to the current section in the wiki, and will check with the team around moving the wiki over to README format to consolidate the rest of the information.